demos/_assets/js/jqm-demos.js

```diff
@@ -315,24 +315,40 @@ $( document ).on( "mobileinit", function() {
 				input = this.element.prev("form").find( "input" );
 
 			if ( e.which === $.ui.keyCode.DOWN ) {
-				if ( this.element.find( "li.ui-button-active" ).length === 0 ) {
-					this.element.find( "li:first" ).toggleClass( "ui-button-active" ).find("a").toggleClass( "ui-button-active" );
+				if ( this.element.find( "li.ui-listview-item-active" ).length === 0 ) {
+					this.element.find( "li:not('.ui-screen-hidden'):first" )
```

Comment:
lets simplify this to `.find( "li ).not( ".ui-screen-hidden" ).first();` we generally prefer methods over selectors